api/src/main/java/org/ehcache/spi/serialization/Serializer.java

```diff
@@ -26,7 +26,17 @@
 
 /**
  * Interface defining the contract used to transform types in a serial form.
- * Implementations must be thread-safe and must define a constructor accepting a single ClassLoader parameter.
+ * Implementations must be thread-safe and must define either or both of the two constructors:
```

Comment:
I agree with @albinsuresh on the location. However, we should make that specific link explicit by referencing here the fact that it applies to usage with the `DefaultSerializationProvider`.